readinglist/views/article.py

```diff
@@ -3,10 +3,10 @@
 from colander import SchemaNode, String
 from pyramid import httpexceptions
 
-from readinglist import errors
-from readinglist.resource import crud, BaseResource, ResourceSchema, TimeStamp
-from readinglist.utils import strip_whitespace
-from readinglist.schema import URL
+from cliquet import errors
+from cliquet.resource import crud, BaseResource, ResourceSchema, TimeStamp
+from cliquet.utils import strip_whitespace
+from cliquet.schema import URL
```

Comment:
We've discussed that and it seems to make sense to have it in cliquet in case we want to reuse it in other projects don't it?